src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesString = sourceString.split(';')
+    .filter(value => value.includes(':'))
+    .map(property => property.trim());
```

Comment:
it's better to move the split method to a new line, in this way you increase the readability of your code (checklist p4)
```suggestion
  const stylesString = sourceString
  	 .split(';')
    .filter(value => value.includes(':'))
    .map(property => property.trim());
```